src/ops/opsconfig.py

```diff
@@ -152,6 +152,15 @@ def ansible_callback_plugins(self):
 
         return os.path.pathsep.join(vars)
 
+    @property
+    def terraform_config_path(self):
+        value = self.config.get('terraform.config_path', None)
```

Comment:
To get rid of the `if` statement can this be rewritten to something like this?

```
def terraform_config_path(self):
    default_path = self.package_dir + '/data/terraform/terraformrc'
    return self.config.get('terraform.config_path', default_path)
```

Reply:
sure :) thanks